test/spec/provider/camunda/element-templates/parts/DescriptionPropsSpec.js

```diff
@@ -180,109 +246,153 @@ describe('element-templates/parts - Description Properties', function() {
-    });
+        // given
+        selectAndGet('Process_1');
 
+        // when
+        clickDropdownMenuItem('elementTemplateDescription', 'element-template-unlink');
 
-    it('should get version as date', function() {
+        // then
+        var process = elementRegistry.get('Process_1'),
+            processBo = getBusinessObject(process);
 
-      // given
-      var template = {
-        version: 1000000000000,
-        metadata: {
-          created: 1000000000000
-        }
-      };
+        expect(processBo.modelerTemplate).not.to.exist;
+      }));
 
-      // when
-      var versionOrDate = getVersionOrDateFromTemplate(template);
 
-      // then
-      expect(versionOrDate).to.match(/Version [0-9]{2}\.[0-9]{2}\.[0-9]{4}/);
+      it('should remove root template', inject(function(elementRegistry, canvas) {
+
+        // given
+        selectAndGet('Process_1');
+
+        // when
+        clickDropdownMenuItem('elementTemplateDescription', 'element-template-remove');
+
+        // then
+        var process = elementRegistry.get('Process_1'),
+            root = canvas.getRootElement(),
+            rootBo = getBusinessObject(root);
+
+        expect(process).not.to.exist;
+
+        expect(root).to.exist;
+        expect(is(root, 'bpmn:Process')).to.be.true;
+
+        expect(rootBo.extensionElements).not.to.exist;
+      }));
+
     });
 
 
-    it('should get version as date', function() {
+    describe('template on root - collaboration', function() {
+      var diagramXML = require('./DescriptionProps-root-collaboration.bpmn'),
+          elementTemplates = require('./DescriptionProps-root.json');
 
-      // given
-      var template = {
-        version: 1000000000000,
-        metadata: {
-          updated: 1000000000000
-        }
-      };
+      beforeEach(bootstrap(diagramXML, elementTemplates));
 
-      // when
-      var versionOrDate = getVersionOrDateFromTemplate(template);
+      it('should unlink root template', inject(function(elementRegistry) {
+
+        // given
+        selectAndGet('Collaboration_1');
+
+        // when
+        clickDropdownMenuItem('elementTemplateDescription', 'element-template-unlink');
+
+        // then
+        var collaboration = elementRegistry.get('Collaboration_1'),
+            collaborationBo = getBusinessObject(collaboration);
+
+        expect(collaborationBo.modelerTemplate).not.to.exist;
+      }));
+
+
+      it('should remove root template', inject(function(elementRegistry, canvas) {
+
+        // given
+        selectAndGet('Collaboration_1');
+
+        // when
+        clickDropdownMenuItem('elementTemplateDescription', 'element-template-remove');
+
+        // then
+        var collaboration = elementRegistry.get('Collaboration_1'),
+            root = canvas.getRootElement(),
+            rootBo = getBusinessObject(root);
+
+        expect(collaboration).not.to.exist;
+
+        expect(root).to.exist;
+        expect(is(root, 'bpmn:Collaboration')).to.be.true;

```

Comment:
There is a few more things that I'd like to see specified, not here but in bpmn-js most likely:

* Does replace keep the old ID? 
* Does root replace keep existing message and error event definitions?
* Does replace keep existing, unknown extension elements?